ase/src/main/java/com/ase/ase/rest/response/TableDataPerDate.java

```diff
@@ -0,0 +1,39 @@
+package com.ase.ase.rest.response;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.util.List;
+
+public class TableDataPerDate {
+
+    private final String date;
+    private final List<HospitalSituationPerDate> situation;
```

Comment:
After the Database SQL Statement and the REST call is implemented, change this to int